WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -85,6 +88,8 @@ class PagesFragment : Fragment() {
     @Inject lateinit var remotePreviewLogicHelper: RemotePreviewLogicHelper
     @Inject lateinit var previewStateHelper: PreviewStateHelper
     @Inject lateinit var progressDialogHelper: ProgressDialogHelper
+    @Inject internal lateinit var uploadActionUseCase: UploadActionUseCase
+    @Inject internal lateinit var uploadUtilsWrapper: UploadUtilsWrapper
```

Comment:
Quick question Jirka. What was your thought process for using the `internal` modifier here? Are these objects gonna be used in other places within this feature set implementation or it has something to do with Dagger? Let me know. 😀

Reply:
I copy pasted the method and didn't notice we don't use "internal" in this fragment.

Dagger doesn't support injection into `private` fields. Best practice is to use the most strict modifier available, which in Kotlin means `internal`. I don't think it matter much since this is not a public API and I don't assume anyone will touch Fragment's fields anyway. But I believe that's the reason why we somewhere use `internal`. Removed in a3bf1776a423d8b30d6e0211e180608a032f4970

Reply:
> Dagger doesn't support injection into private fields. Best practice is to use the most strict modifier available, which in Kotlin means internal. I don't think it matter much since this is not a public API and I don't assume anyone will touch Fragment's fields anyway. But I believe that's the reason why we somewhere use internal. Removed in a3bf177

Yes, good point! Thanks for this explanation!